features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -182,14 +203,24 @@ const EntityDetails = () => {
                                 </div>
                             )}
                             <DataRegionsLegend />
-                            <div className="revenue">
-                                <div className="separator"></div>
-                                <br />
-                                <h2>{i18n.t("entityDetailsScreen:revenue")}</h2>
-                                <CompanyRevenueChart
-                                    annualRevenues={entity.annualRevenues}
-                                />
-                            </div>
+                            {entity?.annualRevenues?.length === 0 ? (
+                                <></>
+                            ) : (
+                                <div className="revenue">
+                                    <div className="separator-unit">
+                                        <div className="partial-separator" />
+                                        <h2>
+                                            {i18n.t(
+                                                "entityDetailsScreen:revenue"
+                                            )}
+                                        </h2>
+                                        <div className="partial-separator" />
```

Comment:
If you're using it always after an `h2`, maybe use `pre` or `post` to include that?

Reply:
🥣